dbt/scripts/run_iasworld_data_tests.py

```diff
@@ -145,11 +151,32 @@ def from_dict(cls, result_dict: typing.Dict) -> "TestResult":
             description=result_dict["description"],
             elapsed_time=result_dict["elapsed_time"],
             failing_rows=result_dict["failing_rows"],
+            notify=result_dict["notify"],
         )
 
     def __repr__(self) -> str:
         return f"TestResult({self.to_dict()!r})"
 
+    @property
+    def details(self) -> str:
+        """Returns a string that can be used to report on the status of
+        this test in detail. Strings are formatted like so:
+
+            - {table_name}: {description} ({status})
+                * {fail1_key1}: {fail1_value1}, {fail1_key2}: {fail1_value2}
+                * {fail2_key1}: {fail2_value1}, {fail2_key2}: {fail2_value2}
```

Comment:
The formatting here is not the most beautiful, but we're limited by the fact that [SNS only supports plaintext](https://stackoverflow.com/a/32243158). We could think about switching to SES if producing a beautiful email is a requirement, though.

Reply:
Let's start simple. IMO the formatting is readable enough.